package.json

```diff
@@ -16,6 +16,7 @@
     "check": "check-peer-dependencies --yarn",
     "npm:auth": "uuip-publish-auth [uuip]",
     "npm:publish": "npm run npm:auth && npm run dist && uuip-publish dist uuip && cd publish && npm publish",
```

Comment:
should we remove these 2 commands?

Reply:
I suppose so. Rost didn't remove it.

Reply:
he doesn't care about cleaning up :) we need to do the thinking beyond what his goal was